test/tests/kernels/graddiv.i

```diff
@@ -0,0 +1,126 @@
+# Grad-div problem using method of manufactured solutions,
+# based on MFEM Example 4.
+
+[Mesh]
+  type = MFEMMesh
+  file = gold/beam-tet.mesh
+  dim = 3
+  uniform_refine = 2
```

Comment:
For the sake of this test, I'd probably just use uniform_refine=1 (or none) for speed, since it's been verified